style.css

```diff
@@ -1 +1,267 @@
+  width: 100%;
+}
+
+header {
+  background-color: var(--second-color);
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.cake-logo {
+  width: 30%;
+}
+
+button {
+  background-color: transparent;
+  border: 0;
+  width: 15vw;
+  margin: 0.5em;
+  cursor: pointer;
+}
+
+.hamburger {
+  width: 100%;
+}
+
+nav {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: flex-end;
+  width: 50%;
+}
+
+.p-header {
+  font-family: "Qwigley", cursive;
+  color: var(--first-color);
+  font-size: 2.2rem;
+}
+
+h1 {
+  font-family: "Dancing Script", cursive;
+  font-size: 3.5rem;
+  margin: 5vw;
+}
+
+.p-main {
+  width: 90vw;
+  margin: 1em auto;
+  font-family: "Parisienne", cursive;
+  text-align: justify;
+  font-size: 1.2rem;
+}
+
+.main-cake {
+  width: 90vw;
+  display: block;
+  margin: 0 auto;
+  border-radius: 0.6em;
+}
+
+.second-section {
+  margin-top: 4vh;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.cake-pictures {
+  width: 40vw;
+  height: 20vh;
+  margin: 3vw;
+}
+
+footer {
+  height: 25vh;
+  background-color: var(--second-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 0%;
+}
+
+.footer-text {
+  font-family: "Dancing Script", cursive;
+  font-size: 1.5rem;
+  padding: 0.5em;
+}
+
+li {
+  list-style-type: none;
+  margin: 0.3em;
+}
+
+a {
+  text-decoration: none;
+}
+
+.social-container {
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+}
+
+.social {
+  border: 1px hidden var(--second-color);
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-shadow: 6px 6px 4px rgba(0, 0, 0, 0.2),
+    -4px -4px 6px rgba(255, 255, 255, 0.3);
+}
+
+.social-media {
+  width: 4rem;
+  padding: 0.7em;
+}
+
+.trademark {
+  font-family: "Dancing Script", cursive;
+  font-size: 1rem;
+  text-align: center;
+}
+
+@media (min-width: 600px) {
+  .hamburger {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: center;
+  }
+
+  .cake-logo {
+    width: 20%;
+    margin: 0.5em;
+  }
+
+  .navbar li {
+    display: inline;
+    color: var(--fourth-color);
+    font-family: "Dancing Script", cursive;
+    font-size: 1.3rem;
+    padding: 0.2em;
+    margin: 0.2em;
+  }
+
+  .p-header {
+    font-size: 1.3rem;
+  }
+
+  .cake-pictures {
+    height: 30vh;
+    border-radius: 0.5em;
+  }
+}
+
+@media (min-width: 1000px) {
+  .navbar li {
+    font-size: 2rem;
+    letter-spacing: 0.1em;
+    transition: transform 0.4s linear;
+  }
+
+  .navbar li:hover {
+    transform: scale(1.3);
+    text-shadow: 2px 2px 3px white;
+  }
+
+  .p-header {
+    font-size: 2rem;
+    letter-spacing: 0.1em;
+  }
+
+  .cake-pictures {
+    width: 20vw;
+    height: 30vh;
```

Comment:
Watch out here, don't set a width AND a height to your image, it's causing distortion. You can wrap each image with a div which can have width and height, and then set the `img` to be `width: 100%` but don't set a height. The img will then naturally fill the width of the wrapper div, but without losing its aspect ratio